javascript/javascript1/week4/h4.js

```diff
@@ -0,0 +1,97 @@
+let userName = "";
+const toDoList = [];
+
+function getReply(command) {
+    if (typeof command !=='string' || command === "") {
+        return `Error! Not a valide command`;
+    } if (command === "Hello my name is Violeta" && userName === "") {
+        userName = "Violeta";
+        return `Nice to meet you ${userName}`;
+    } else if (command === "Hello my name is Violeta" && userName === ""){
+        userName="";
+        return `Your name is already added`;
+    } if (command === "What is your name?" && userName != "") {
+        return `Your name is ${userName}`;
+   
+    }  else if ( command === "What is your name" && userName === "") {
+        return 'Your name is not mentioned';
+    }  if (command === "Add fishing to my to do list" ) {
+        toDoList.push("fishing");
+        return "Fishing added to your to do list";
+    }  if (command === "Add singing in the shower to my to do list") {
+        toDoList.push("sing in the shower");
+        return "Singing in the shower added to your to do list";
+    } if (command === "Remove fisihing from my to do list") {
+        const indexOftoDolistRemove = toDoList.indexOf("fishing");
+        toDoList.splice(indexOftoDolistRemove, 1);
+        return "Fishing removed from your to do list";
```

Comment:
Similar to my previous comment. It would be great if you can write a function which would check which todo is written in the command and it would remove that particular todo from `todoList`. You would call the function in all cases where the command is `Remove ____ from my todo.` You should also check if the particular todo is in the `todoList`.